fix: update links (#17584)
MystenLabs/sui: docs/content/guides/developer/cryptography/multisig.mdx

description: Guide on how to create a multisig transaction and then submit it against a local network using the Sui CLI.
---

The following steps demonstrate how to create a multisig transaction and then submit it against a local network using the [Sui CLI](/references/cli.mdx). A transaction can be the transfer of an object, the publish or upgrade of a package, the payment of SUI, and so on. To learn how to set up a local network, see [Connect to a Local Network](/guides/developer/getting-started/local-network.mdx). 

> To learn more how to create multisig address and create multisig transaction from Typescript SDK, see [this page](https://sdk.mystenlabs.com/typescript/cryptography/multisig) for details. 

Reviewer: ```suggestion
To learn more about how to create multisig addresses and create multisig transactions using the TypeScript SDK, see the [SDK documentation](https://sdk.mystenlabs.com/typescript/cryptography/multisig) for details. 
```

## Related links

- [Multisig](../../../concepts/cryptography/transaction-auth/multisig.mdx): Definition of multisig transactions on the Sui network and how to use it with CLI.

> - [Multisig Typescript SDK](https://sdk.mystenlabs.com/typescript/cryptography/multisig): How to create multisig address and sign transactions using the Sui SDK.

Author: not sure how to link the sdk page's original page, feel free to fix @ronny-mysten

Reviewer: My guess is you've never not figured something out correctly. This case included ;)

> - [Multisig Typescript SDK](https://sdk.mystenlabs.com/typescript/cryptography/multisig): How to create multisig address and sign transactions using the Sui SDK.

Reviewer: ```suggestion
- [Multisig Typescript SDK](https://sdk.mystenlabs.com/typescript/cryptography/multisig): How to create multisig addresses and sign transactions using the Sui SDK.
```
